docs/tutorials/01_electronic_structure.ipynb

```diff
@@ -119,192 +128,138 @@
     "* The Jordan-Wigner 'jordan_wigner' mapping (über das paulische äquivalenzverbot. In The Collected Works of Eugene Paul Wigner (pp. 109-129). Springer, Berlin, Heidelberg (1993)).\n",
     "* The Parity 'parity' (The Journal of chemical physics, 137(22), 224109 (2012))\n",
     "* The Bravyi-Kitaev 'bravyi_kitaev' (Annals of Physics, 298(1), 210-226 (2002))\n",
-    "* The Bravyi-Kitaev Super Fast 'bksf' (Annals of Physics, 298(1), 210-226 (2002))\n",
     "\n",
     "The Jordan-Wigner mapping is particularly interesting as it maps each Spin Orbital to a qubit (as shown on the Figure above). \n",
     "\n",
-    "Here we set up an object which contains all the information about any transformation of the fermionic Hamiltonian to the qubits Hamiltonian. In this example we simply ask for the Jordan-Wigner mapping. "
+    "Here we set up the Electronic Structure Problem to generate the Second quantized operator and a qubit converter that will map it to a qubit operator."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
-    "from qiskit.chemistry.transformations import (FermionicTransformation, \n",
-    "                                              FermionicTransformationType, \n",
-    "                                              FermionicQubitMappingType)\n",
-    "\n",
-    "fermionic_transformation = FermionicTransformation(\n",
-    "            transformation=FermionicTransformationType.FULL,\n",
-    "            qubit_mapping=FermionicQubitMappingType.JORDAN_WIGNER,\n",
-    "            two_qubit_reduction=False,\n",
-    "            freeze_core=False)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "If we now transform this Hamiltonian for the given driver defined above we get our qubit operator:"
+    "from qiskit_nature.problems.second_quantization.electronic import ElectronicStructureProblem\n",
+    "from qiskit_nature.operators.second_quantization.qubit_converter import QubitConverter\n",
```

Comment:
Are these two entities `ElectronicStructureProblem` and `QubitConverter` directly importable from `from qiskit_nature.operators.second_quantization` rather than having to import directly from the specific module - I think they should be an most likely this is where they will show in the docs too